src/main/java/com/mraof/minestuck/entity/underling/OgreEntity.java

```diff
@@ -33,14 +36,7 @@ public static AttributeModifierMap.MutableAttribute ogreAttributes()
 				.add(Attributes.KNOCKBACK_RESISTANCE, 0.4).add(Attributes.MOVEMENT_SPEED, 0.22)
 				.add(Attributes.ATTACK_DAMAGE, 6);
```

Comment:
Add a TODO to change the animation so it looks less like it is running(should be changed to look like its walking), currently there is a big visual discrepancy between how it moves and how it looks like it moves